website/content/docs/v0.7/Guides/upgrading-talos.md

```diff
@@ -2,15 +2,110 @@
 title: Upgrading Talos
 ---
 
-## Talos
+## Overview
 
-In an effort to create more production ready clusters, Talos will now taint control plane nodes as unschedulable.
-This means that any application you might have deployed must tolerate this taint if you intend on running the application on control plane nodes.
+The upgrade process for Talos, like everything else, begins with an API call.
+This call tells a node the installer image to use to perform the upgrade.
+Each Talos version corresponds to an installer with the same version, such that the
+version of the installer is the version of Talos which will be installed.
 
-Another feature you will notice is the automatic uncordoning of nodes that have been upgraded.
-Talos will now uncordon a node if the cordon was initiated by the upgrade process.
+Because Talos is image based, even at run-time, upgrading Talos is almost
+exactly the same set of operations as installing Talos, with the difference that
+the configuration is already in place.
```

Comment:
Maybe add to this a bit? "configuration is already in place and ..." IDK the exact wording, but that the system us fully booted. IDK. Could live without this idea too :smile: .